include/native/wsi/native_sdl2.h

```diff
@@ -1,7 +1,13 @@
 #include <cstdint>
 #include <windows.h>
 
+#ifdef DXVK_PLATFORM_DARWIN
+#include <SDL.h>
+#include <SDL_vulkan.h>
```

Comment:
Why is this needed? Can we solve it in the build script? :(

Reply:
Interestingly the recommended way is using `#include "SDL.h"` according to FAQ https://wiki.libsdl.org/FAQDevelopment so this way is more correct.

Using SDL2/... didn't work on MacOS, without SDL2/... works on my Linux and MacOS after using dependency() meson call as in MacOS it can't find SDL2 when using find_library(), it also seems the recommended way to search dependencies.

I merged darwin and linux branches in meson for dependencies.

Reply:
Right, the other advantage is that `find_library()` won't let you detect a custom compiled version of SDL2 added to your development environment via PKG_CONFIG_PATH.

Using dependency() also means you can use https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section